lua/starfall/libs_cl/render.lua

```diff
@@ -1676,6 +1682,22 @@ function render_library.renderView(tbl)
 	data.isRendering = true
 end
 
+hook.Add("ShouldDrawLocalPlayer", "SF_DrawLocalPlayerInRenderView", function()
+	if renderingView and drawViewerInView then
+		cam.Start3D()
```

Comment:
What's this for?

Reply:
For drawing the local player in the renderview. By default, it's disabled. cam.Start3D() and cam.End3D() are a workaround for a gmod bug http://wiki.garrysmod.com/page/GM/ShouldDrawLocalPlayer.